### PR TITLE
🌻 Sunflowers 🌻

### PR DESCRIPTION
**Pull request by Azrun**
## About the PR 
Leverages the existing sprite to have... SUNFLOWERS.  Seeds will automatically show up in the vendor.
If you take the flower and use them in your hands, hit them, or you hit something with them the seeds will pop out.  Eat the seeds for a chance to get seeds you can grow.
🌻🌻🌻🌻🌻🌻

## Why's this needed? 
We already have the sprite, might as well allow it for decorative purposes.

I would love for these to be around when bouquets happen.

## Changelog 

```changelog
(u)Azrun
(+)Sunflowers can now be grown! Whack 'em for sunflower seeds!
```

### CODE REVIEW

File: code/modules/food_and_drink/plants.dm [hunk 1]
```diff
@@ -1455,6 +1455,31 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 		..()
 		reagents.add_reagent("milk", 5)
 
+/obj/item/reagent_containers/food/snacks/plant/sunflower
+	name = "sunflower seeds"
+	crop_suffix = " seeds"
+	desc = "Even though the coffee beans are seeds, they are referred to as 'beans' because of their resemblance to true beans.."
```
Comment (NibChocolateeny) [inline code/modules/food_and_drink/plants.dm:1461]: dont forget to change descrpition!

File: code/modules/food_and_drink/plants.dm [hunk 2]
```diff
@@ -1455,6 +1455,32 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
...
+	bites_left = 3
+	heal_amt = 0.2
+	food_color = "#695b59"
+	food_effects = list("food_refreshed")
+
+	HYPsetup_DNA(var/datum/plantgenes/passed_genes, var/obj/machinery/plantpot/harvested_plantpot, var/datum/plant/origin_plant, var/quality_status)
+		switch(quality_status)
+			if("jumbo")
+				src.heal_amt = 0.5
+				src.bites_left *= 1
```
Comment (Tarmunora) [inline code/modules/food_and_drink/plants.dm:1473]:
```suggestion
```

**Review by Tarmunora — CHANGES_REQUESTED** [review]
Any reason to not just instantiate the seeds in `disperse_seeds()`?

**Comment by Azrun** [comment]
Is there any reason not to just instantiate the seeds in `disperse_seeds () '?

I think it would necessitate the need to recalculate `quality_status 'and I wanted to keep the general flow consistent with how it would normally behave with the reference to the plantpot, but those aren't applicable with the override.

I'll poke again.  Also I was investigating a component that could seedify things like glass...

**Pull request merged**